cumulus/parachains/runtimes/assets/ethereum_standards/src/erc20.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Solidity ERC20 interface.
+
+use alloy_core::sol;
+
+sol! {
+	interface IERC20 {
```

Comment:
I see only a minority of the [required methods](https://eips.ethereum.org/EIPS/eip-20) here, is the idea to add them later?

Reply:
I would assume only the functions needed for the transactor where added. But when calling the crate "ethereum_standard" it would make sense to just copy over the full interface. Even though if we never call those functions. For re-useablity.

Reply:
Yeah I thought the exact same thing: This looks re-usable (and useful!) but incomplete.

Reply:
I recommend just copying over the interface .sol file verbatim and then just use:
```
alloy::sol!("ERC20.sol");
```

Nicer than embedding it into Rust I think for this case.

Reply:
yeah see my PR here #8554, you can copy it from here, looks like the macro has some issue with doc comment I had to reformat it a bit

also why does this live under cumulus ?

Reply:
This cumulus/substrate/polkadot devision is kinda only historical. But yeah it should probably be in substrate.

Reply:
I can move it to substrate, doesn't bother me